### PR TITLE
treewide: Remove Python 2 plugin/bindings/etc from non-Python packages

### PR DESCRIPTION
**Pull request by jefferyto**
Maintainers:
* @ClaymorePT - boost
* @haraldg - elektra
* @blogic, @nxhack - libmraa, libupm
* me, @commodo - libsearpc, seafile-ccnet, seafile-server
* @thess - libtalloc
* @Robby- (interested user) - freeradius3
* @dangowrt - i2c-tools 

Compile tested: armvirt-64, 2020-03-29 snapshot sdk
Run tested: none

Description:
This removes Python 2 plugin/bindings packages (and other Python 2-related files) from non-Python packages.

### CODE REVIEW

**Comment by jefferyto** [comment]
The CircleCI test will most likely fail because it usually fails to compile boost, and also libmraa currently has a [compile error](https://downloads.openwrt.org/snapshots/faillogs/x86_64/packages/libmraa/compile.txt) (I saw this when compiling locally as well).

**Comment by ClaymorePT** [comment]
@jefferyto Thank you for taking the time and I'm sorry for not having done this at the beginning of the year. Just one question. Since Python2 is now removed, shouldn't the python3 library be now called python, instead of python3?

File: net/seafile-server/Makefile [hunk 1]
```diff
@@ -174,9 +171,6 @@ define Package/seafile-server/install
 		$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/$$$$bin $(1)/usr/libexec/ ; \
 	done
 
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/python/seafile/*.py $(1)$(PYTHON_PKG_DIR)/seafile/
-	$(INSTALL_DATA) $(PKG_BUILD_DIR)/python/seaserv/*.py $(1)$(PYTHON_PKG_DIR)/seaserv/
```
Comment (commodo) [inline net/seafile-server/Makefile:178]: should this be `PYTHON3_PKG_DIR` ?
or do we need it?

File: net/seafile-ccnet/Makefile [hunk 2]
```diff
@@ -74,12 +73,9 @@ CONFIGURE_ARGS += \
 	--enable-python \
 	--with-mysql="$(STAGING_DIR)/usr/bin/mysql_config"
 
-# Include Python 2 bindings in this package to maintain compatibility with previous versions
 define Package/seafile-ccnet/install
 	$(INSTALL_DIR) $(1)/usr/{lib,libexec}
-	$(INSTALL_DIR) $(1)$(PYTHON_PKG_DIR)/ccnet
```
Comment (commodo) [inline net/seafile-ccnet/Makefile:80]: should this be `PYTHON3_PKG_DIR`?
or do we need it?

**Comment by jefferyto** [comment]
@ClaymorePT In this PR I've tried to remove all traces of Python 2 with the minimal amount of changes (the boost commit probably does more than absolutely necessary already). If you'd like to rename the `python3` library to `python` I think this can done later.

Most/all other references to Python (including the Python interpreter) will continue to be `python3` (or some variation that includes the "3"), as no decision about a global renaming of `python3` to `python` has been made, so I don't think keeping the Boost library as `python3` for now is a problem.

File: net/seafile-server/Makefile (hunk 1 above)
Comment (jefferyto) [reply net/seafile-server/Makefile:178]: I added a separate [python3-seafile-server](https://github.com/openwrt/packages/blob/76bd346085ce50c05c27ea1d6fa1452e015f36d7/net/seafile-server/Makefile#L109) package where those files are [placed](https://github.com/openwrt/packages/blob/76bd346085ce50c05c27ea1d6fa1452e015f36d7/net/seafile-server/Makefile#L212-L213) in `PYTHON3_PKG_DIR`.

File: net/seafile-ccnet/Makefile (hunk 2 above)
Comment (jefferyto) [reply net/seafile-ccnet/Makefile:80]: Similar to seafile-server, there is now a [python3-seafile-ccnet](https://github.com/openwrt/packages/blob/76bd346085ce50c05c27ea1d6fa1452e015f36d7/net/seafile-ccnet/Makefile#L56) package.

File: net/seafile-server/Makefile (hunk 1 above)
Comment (commodo) [reply net/seafile-server/Makefile:178]: ack

File: net/seafile-ccnet/Makefile (hunk 2 above)
Comment (commodo) [reply net/seafile-ccnet/Makefile:80]: ack

**Pull request merged**